src/any_agent/frameworks/any_agent.py

```diff
@@ -104,13 +104,23 @@ def create(
         tracing: TracingConfig | None = None,
     ) -> AnyAgent:
         """Create an agent using the given framework and config."""
-        return asyncio.get_event_loop().run_until_complete(
-            cls.create_async(
-                agent_framework=agent_framework,
-                agent_config=agent_config,
-                tracing=tracing,
+        try:
+            return asyncio.get_event_loop().run_until_complete(
+                cls.create_async(
+                    agent_framework=agent_framework,
+                    agent_config=agent_config,
+                    tracing=tracing,
+                )
+            )
+        except RuntimeError:
```

Comment:
Is there a more specific error we can check?
Also, can we factor this out, like....

```
def exec(callable):
        try:
            return asyncio.get_event_loop().run_until_complete(
                callable()
            )
        except RuntimeError:
            # No running event loop, we can use asyncio.run()
            return asyncio.run(
                callable()
            )
```

_Also_, is there some intermediate condition that would not be covered? I can't think of any, just in case.